packages/api-explorer/__tests__/form-components/CheckboxWidget.test.jsx

```diff
@@ -0,0 +1,16 @@
+const React = require('react');
+const { shallow } = require('enzyme');
+const Checkbox = require('../../src/form-components/CheckboxWidget');
+
+test('should not render description', () => {
+  const checkbox = shallow(
+    <Checkbox
+      id="id"
+      label="label"
+      required
+      onChange={() => {}}
+      schema={{ description: 'description' }}
+    />,
+  );
+  expect(checkbox.html()).not.toMatch(/description/);
```

Comment:
Question the .not.toMatch(/description/) means the string description shouldn't show up in the snapshot correct?

Reply:
I removed the snapshot test. It was simple enough to do it without snapshots